addons/mail/static/src/components/discuss/tests/discuss_sidebar_tests.js

```diff
@@ -138,6 +138,22 @@ QUnit.test('sidebar find shows channels matching search term even when user is m
     );
 });
 
+QUnit.test('sidebar channels should ordered alphabetically', async function (assert) {
+    assert.expect(1);
+
+    this.data['mail.channel'].records.push({ id: 19, name: "Xyz" },
+                                        { id: 20, name: "abc" },
+                                        { id: 21, name: "Abc" },
+                                        { id: 22, name: "Xyz" });
```

Comment:
I thought I sent this comment earlier, but don't try to align indent like this, just
```suggestion
    this.data['mail.channel'].records.push(
        { id: 19, name: "Xyz" },
        { id: 20, name: "abc" },
        { id: 21, name: "Abc" },
        { id: 22, name: "Xyz" },
    );
```

Reply:
@seb-odoo Changes Done.